src/main/kotlin/systems/ajax/englishstudytelegrambot/service/WordServices.kt

```diff
@@ -0,0 +1,88 @@
+package systems.ajax.englishstudytelegrambot.service
+
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.server.ResponseStatusException
+import systems.ajax.englishstudytelegrambot.dto.AudioForWordResponse
+import systems.ajax.englishstudytelegrambot.dto.DefinitionOfWordResponse
+import systems.ajax.englishstudytelegrambot.dto.ExampleOfWordResponse
+import systems.ajax.englishstudytelegrambot.dto.PronunciationOfWordResponse
+import systems.ajax.englishstudytelegrambot.dto.GettingPartOfAdditionalInfoAboutWord
+import systems.ajax.englishstudytelegrambot.entity.AdditionalInfoAboutWord
+import kotlin.reflect.KSuspendFunction1
+
+interface AdditionalInfoAboutWordService {
+    suspend fun findAdditionInfoAboutWord(wordSpelling: String): AdditionalInfoAboutWord
+}
+
+@Service
+class AdditionalInfoAboutWordServiceImpl(private val webClient: WebClient) : AdditionalInfoAboutWordService {
+
+    @Value("\${wordnik.api.key}")
+    lateinit var wordnikAPIKey: String
+
+    @Value("\${link.audio}")
+    lateinit var audioLink: String
+
+    @Value("\${link.definition}")
+    lateinit var definitionLink: String
+
+    @Value("\${link.example}")
+    lateinit var exampleLink: String
+
+    @Value("\${link.pronunciation}")
+    lateinit var pronunciationLink: String
+
+    val log = LoggerFactory.getLogger(this::class.java)
```

Comment:
```suggestion
    companion object {
        val log = LoggerFactory.getLogger(this::class.java)
    }
```

Note: companion object must be in the end of class